src/parkapi_sources/converters/vrs_p_r/vrs_p_r.py

```diff
@@ -45,43 +42,16 @@ class VrsParkAndRidePushConverter(NormalizedXlsxConverter):
         # Other opening times are there, but not parsable
     }
 
-    def handle_xlsx(self, workbook: Workbook) -> tuple[list[StaticParkingSiteInput], list[ImportParkingSiteException]]:
-        worksheet = workbook.active
-        mapping: dict[str, int] = self.get_mapping_by_header(next(worksheet.rows))
-        row_values = [cell.value for cell in next(worksheet.rows)]
+    def map_row_to_parking_site_dict(self, mapping: dict[str, int], row: list[Cell], **kwargs) -> dict[str, Any]:
+        parking_site_dict = super().map_row_to_parking_site_dict(mapping, row, **kwargs)
+        column_names = kwargs.get('column_names', [])
```

Comment:
You can set `column_names` directly in the method definition, because you actually use it here